File: CheckifArrayisSorted.java
public class CheckifArrayisSorted {
    public static void main(String[] args) {
        int[] arr = {1, 5, 8, 9, 10, 15};
        boolean isSorted = true;

        for(int i = 1; i < arr.length; i++) {// simple brute force check agla element bada hai ya nahi
            if(arr[i] < arr[i - 1]) {
                isSorted = false;
                break;
            }
        }

        System.out.println(isSorted ? "YES" : "NO");
    }
}

File: python/CheckifArrayisSorted.py
# Check if array is sorted in ascending order
arr = [1, 5, 8, 9, 10, 15]
is_sorted = True
for i in range(1, len(arr)):
    if arr[i] < arr[i - 1]:
        is_sorted = False
        break
print("YES" if is_sorted else "NO")
